keras_cv/models/segmentation/deeplab.py

```diff
@@ -13,152 +13,116 @@
 # limitations under the License.
 
 import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
 
 from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
-from keras_cv.models.resnet_v2 import ResNetV2
-from keras_cv.models.weights import parse_weights
+from keras_cv.models import utils
 
-BACKBONE_CONFIG = {
-    "ResNet50V2": {
-        "stackwise_filters": [64, 128, 256, 512],
-        "stackwise_blocks": [3, 4, 6, 3],
-        "stackwise_strides": [1, 2, 2, 2],
-        "stackwise_dilations": [1, 1, 1, 2],
```

Comment:
Did `stackwise_dilations` got removed?

Reply:
Yup - we don't need the config anymore since the backbone is being supplied by the user instead of being built within DeepLabV3.
We'll want the user to supply dilations as well. 

I forgot to add them in the training script after removing here - that's updated now